docs/LearnCalico.md

```diff
@@ -0,0 +1,351 @@
+waiting for changes in the etcd datastore.
+
+![calicoctl node](diagrams/calicoctl_node.png)
+
+calico-docker allows you to configure other BGP topologies as well, such as a 
+mesh of route reflectors or per-node BGP.  Visit the calico-docker [BGP 
+page](https://github.com/projectcalico/calico-docker/blob/master/docs/bgp.md) 
+for more details.
+
+### Create Containers on Host
+
+Two containers are created using the orchestrator (such as running
+`docker run --name=workload_A -tid ubuntu`).  The workloads have not been added 
+to Calico networking and no security policy has been configured at this point, 
+so they do not yet have connectivity or Calico-assigned IP addresses.  
+
+The workloads do not have connectivity between one another, but they can access 
+the internet via the docker bridge interface. None of the Calico processes have 
+performed any new actions at this point.
+
+![docker run](diagrams/docker_run.png)
+
+**Note**: Docker's libnetwork plugin works a bit differently than this 
+default-networking example. When using Calico with libnetwork, this step and the 
+next three steps in the process all occur at the same time.  The container and 
+profile are created simultaneously, the container is added to Calico networking, 
+and the profile is set on the container.  Continue reading to learn more about 
+what happens under the covers.
+
+### Add Containers to Calico Networking
+
+The workloads are added to Calico networking by calling:
+
+    calicoctl container add <container_id> <ip_address>
+
+The `<container_id>` can be either the name of the container or the container's 
+workload id.  The `<ip_address>` must be an address that falls within a 
+configured IP pool.  The default IPv4 pool for calico is `192.168.0.0/16`. 
+You can view pools by running `calicoctl pool show`. To add a new pool, run 
+`calicoctl pool add <cidr>` or `calicoctl pool range add <start_ip> <end_ip>`. 
+Etcd stores pools as `/calico/v1/ipam/v4/pool/192.168.0.0-16`, where assigned 
+IPs in a pool are stored as: `/calico/v1/ipam/v4/assignment/192.168.0.0-16/192.168.1.1`.
+
+When a workload is added to Calico, the `calicoctl` tool checks the etcd 
+datastore to confirm that the ip address passed in falls under a previously 
+defined IP pool. It then creates a veth pair on the host for the new endpoint, 
+storing one end in the host namespace and moving the other end into the 
+container.  After successfully creating the veth pair, `calicoctl` saves the 
+workload id, the endpoint id, the endpoint's assigned IP, the host's IP, the 
+container's new veth MAC address, and the name of the veth on the host's end 
+to the etcd datastore as:
+
+    # KEY
+    /calico/v1/host/Host1/workload/docker/<workload_id>/endpoint/<endpoint_id>
+    
+    #VALUE
+    {"ipv6_gateway": null, "state": "active", "name": "<host_veth_name>", 
+     "ipv4_gateway": "<host_ip>", "ipv6_nets": [], "profile_ids": [], 
+     "mac": "<container_mac>", "ipv4_nets": ["<assigned_ip>"]}
```

Comment:
Two suggestions:

1. For consistency, there should be a space between `#` and `VALUE`
2. Would it be worth indenting the JSON for readability?